src/index.js

```diff
@@ -311,7 +311,15 @@ class WarmUp {
     const warmUpFunction = `"use strict";
 
 /** Generated by Serverless WarmUp Plugin at ${new Date().toISOString()} */
-const aws = require("aws-sdk");
+let aws;
+try {
+  const AWSXRay = require("aws-xray-sdk-core");
```

Comment:
How is  `aws-xray-sdk-core` installed in the lambda?

Reply:
The same way aws-sdk is: by the user installing it in the project they add this plugin to.

Reply:
That's not how this plugin works.
`aws-sdk` is always present in the Nodejs lambdas without the user having to install it.

So, as I suspected, this would always throw and use the standard uninstrumented library.

Are you planning to test this in any project of yours?
I have some ideas about how this could work and could create a branch for you to test.
Unfortunately, I don't have time to test this myself at the moment on an actual AWS deployment.

Reply:
Yes, I've tested it. :)